omnisafe/algorithms/model_based/base/ensemble.py

```diff
@@ -466,8 +581,9 @@ def train(
         holdout_inputs, holdout_labels = inputs[:num_holdout], labels[:num_holdout]
         self._ensemble_model.scaler.fit(train_inputs)
 
+        train_mse_losses = []
```

Comment:
will this change the original code behavior?

Reply:
I have considered it. Because original function use the `train_mse_loss` in the for loop, that will raise `unbound` error. I just log the total training loss to log, but still use loss in loop to determine whether to break. The performance curve seems it does not change original code behavior.